configs/module/network/ssast.yaml

```diff
@@ -0,0 +1,34 @@
+model:
+  _target_: birdset.modules.models.ssast.ASTModel
+  load_pretrained_mdl_path: ${paths.root_dir}/models/ssast/SSAST-Base-Patch-400.pth
+  model_size: base
+  label_dim: ${datamodule.dataset.n_classes}
+  pretrain_stage: False
+  fstride: 16 # The stride of patch spliting on the frequency dimension, for 16*16 patchs, fstride=16 means no overlap, fstride=10 means overlap of 6.
+  tstride: 16 # The stride of patch spliting on the time dimension, for 16*16 patchs, tstride=16 means no overlap, tstride=10 means overlap of 6.
+  input_tdim: 500 # The number of time frames of the input spectrogram.
+  
+
+
+
+model_name: ssast
+model_type: vision
+torch_compile: False
+sampling_rate: 32_000
+normalize_spectrogram: True
+normalize_waveform: "instance_normalization"
+embedding_size: 768 # Needs to be adjusted if the other params are changed
```

Comment:
@XgamerTV When what parameters change?

Reply:
When using different model sizes?

Reply:
The parameters are now explained in ssast.yaml and the code itself